src/encoded/static/components/dataset.js

```diff
@@ -225,7 +225,7 @@ class AnnotationComponent extends React.Component {
                 </Panel>
 
                 {/* Display the file widget with the facet, graph, and tables */}
-                <FileGallery context={context} encodevers={encodevers} />
+                <FileGallery context={context} encodevers={encodevers} showReplicateNumber={false}  />
```

Comment:
I think there are two spaces after 
```
showReplicateNumber={false}
```
The ticket is already merge ready, but I guess we should fix it?

Reply:
Probably should be fixed. Looks like it causes an eslint error. @zoldello Do you have eslint configured in Atom in a way that uses the encoded .eslintrc file?

Reply:
Sounds good.  I'll mark this in PR as review requested.  I don't think we need a new QA though, right?

Reply:
oh, it showed up in codacy.  I should look at that more